frontend/src/config/jordan/layers.json

```diff
@@ -4022,18 +4022,31 @@
     "input_layers": [
       {
         "id": "spi_1m",
-        "weight": 0.4,
-        "interval": "1-month"
+        "importance": 0.4,
+        "key": [
```

Comment:
The keys need to be confirmed. Available keys are listed in https://github.com/WFP-VAM/hip-analysis/blob/dev/docs/reference/datasources.md

Reply:
@wadhwamatic could you sync up with the Jordan team and confirm the CDI configuration?
This is what I inferred for now but it's probably off for some of the layers.

```
    "input_layers": [
      {
        "id": "spi_1m",
        "importance": 0.4,
        "key": [
          "CHIRPS",
          "R1S_DEKAD"
        ],
        "aggregation": "last_dekad"
      },
      {
        "id": "lst_anomaly",
        "importance": 0.3,
        "key": [
          "MODIS",
          "MYD11C2_TDD_DEKAD"
        ],
        "aggregation": "last_dekad"
      },
      {
        "id": "ndvi_dekad_anomaly",
        "importance": 0.3,
        "key": [
          "MODIS",
          "NDVI_smoothed_5KM"
        ],
        "aggregation": "average",
        "invert": "True"
      }
    ],
```